### PR TITLE
Dark Mode (WIP)

### PR DESCRIPTION
**Pull request by aliceisjustplaying**
Unfucked the rebase and branches, now I can get back to work

### CODE REVIEW

File: apps/expo/src/app/_layout.tsx [hunk 1]
```diff
@@ -109,19 +112,19 @@ export default function RootLayout() {
...
               headerBackTitle: "",
               fullScreenGestureEnabled: true,
               headerStyle: {
-                backgroundColor: "#fff",
+                backgroundColor: colorScheme === "light" ? "#fff" : "#000",
               },
             }}
           />
-          <StatusBar />
+          <StatusBar style="auto" />
```
Comment (mozzius) [inline apps/expo/src/app/_layout.tsx:127]: don't think there needs to be 2 `<StatusBar>`s

File: apps/expo/src/components/rich-text.tsx [hunk 2]
```diff
@@ -58,7 +58,11 @@ export const RichText = ({ text, facets, size = "base" }: Props) => {
       } else {
         parts.push({
           text: segment.text,
-          component: segment.text,
+          component: (
+            <Text className="text-neutral-500 dark:text-neutral-50">
+              {segment.text}
+            </Text>
+          ),
```
Comment (mozzius) [inline apps/expo/src/components/rich-text.tsx:65]: Text is way too light on light mode. I'd be inclined to removed all instances of `text-neutral-500` and just use the automatic colour

**Review by mozzius — COMMENTED** [review]
Just some comments. Main one is the text colour for light mode is now too pale - can we just leave it as the default colour?

File: apps/expo/src/app/(tabs)/skyline.tsx [hunk 3]
```diff
@@ -36,22 +36,25 @@ export default function Timeline() {
...
       switch (mode) {
-        case "popular":
+        case "popular": {
           const popular = await agent.app.bsky.unspecced.getPopular({
             cursor: pageParam as string | undefined,
           });
           if (!popular.success) throw new Error("Failed to fetch feed");
           return popular.data;
-        case "following":
+        }
```
[the remark this answers is not in the document]
Comment (aliceisjustplaying) [reply apps/expo/src/app/(tabs)/skyline.tsx:45]: eslint complains about it, i added a few stricter rules to the ruleset

File: apps/expo/src/app/_layout.tsx (hunk 1 above)
Comment (aliceisjustplaying) [reply apps/expo/src/app/_layout.tsx:127]: ok this is crazy but if i remove one of them the whole thing breaks, need to debug

File: apps/expo/src/app/(tabs)/skyline.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (mozzius) [reply apps/expo/src/app/(tabs)/skyline.tsx:45]: Happy to adopt this across the codebase then

File: apps/expo/src/app/_layout.tsx (hunk 1 above)
Comment (mozzius) [reply apps/expo/src/app/_layout.tsx:127]: haha ok, I do love a load-bearing useless component

**Comment by mozzius** [comment]
You can set the background color using this: https://expo.github.io/router/docs/migration/react-navigation/themes/

**Comment by mozzius** [comment]
lol

```ts
import { DefaultTheme, DarkTheme } from '@react-navigation/native';
```

**Comment by lucasgelfond** [comment]
fastest bsky dev in the west

**Pull request merged**